content/docs/reconciliation.md

```diff
@@ -4,32 +4,32 @@ title: Reconciliation
 permalink: docs/reconciliation.html
 ---
 
-React provides a declarative API so that you don't have to worry about exactly what changes on every update. This makes writing applications a lot easier, but it might not be obvious how this is implemented within React. This article explains the choices we made in React's "diffing" algorithm so that component updates are predictable while being fast enough for high-performance apps.
+React cung cấp một API khai báo để bạn không phải lo lắng về những thay đổi chính xác trên mỗi bản cập nhật. Điều này làm cho việc viết ứng dụng trở nên dễ dàng hơn rất nhiều, nhưng có thể không rõ ràng điều này được thực hiện như thế nào trong React. Bài viết này giải thích các lựa chọn mà chúng tôi đã thực hiện trong thuật toán "khác biệt" của React để các bản cập nhật thành phần có thể dự đoán được trong khi vẫn đủ nhanh cho các ứng dụng hiệu suất cao.
```

Comment:
I recommend to update some point for more readable and easy for understanding with Vietnamese users:
`cung cấp một API khai báo...` ---> `cung cấp một API tự động xác định những nơi bị thay đổi sau những lần cập nhật`
`nhưng có thể không rõ ràng điều này được thực hiện như thế nào...` --> `nhưng nếu bạn muốn tìm hiểu thêm về cách React xác định những thay đổi trong DOM như thế nào.`
`Bài viết này sẽ giải thích cách React sử dụng thuật toán "diffing" để đưa ra quyết định cập nhật components của bạn một cách có kiểm soát và đảm bảo hiệu năng (performance) của ứng dụng`